app/forms/DateOfBusinessTransferFormProvider.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import identifiers.DateOfBusinessTransferId
+import models.DateFormElement
+import play.api.data.Form
+import play.api.data.Forms._
+import utils.TimeMachine
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class DateOfBusinessTransferFormProvider @Inject()(timeMachine: TimeMachine) extends FormErrorHelper with Mappings {
+
+  val dateOfBusinessTransferDate = s"${DateOfBusinessTransferId}Date"
```

Comment:
The name start and ends with `date` which is bit weird but I am just following the existing convention (i.e. form name + field name). Suggestions are welcome.